src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/search-button.html

```diff
@@ -1,14 +1,5 @@
 {# Search page has its own UI / UX for search #}
-<!-- A button that will trigger a search field to be displayed on click. -->
-{% set containerClass="search-button__search-container" %}
-
 <!-- An overlay that will expand in the background and cause the search to disappear when clicked -->
```

Comment:
this comment, and the comment on line 1, both seem outdated / irrelevant now: the overlay was removed from this file, and the fact that the search page has its own UI for search matters, but doesn't matter *here*

Reply:
comments removed! good catch